build/build.ps1

```diff
@@ -150,7 +152,17 @@ if (!$SkipBuild)
     $nugetProc.WaitForExit()
 
     $solutionDir = Join-Path $pwd "..\source"
-    $msbuildPath = "c:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe";
+
+    if (-not (Get-Command -Name $VSWHERE_CMD -Type Application -ErrorAction Ignore))
+    {
+        $VSWHERE_CMD = (Join-Path $solutionDir "packages\vswhere.2.6.7\tools\vswhere.exe")
+        if (-not (Get-Command -Name $VSWHERE_CMD -Type Application -ErrorAction Ignore))
+        {
+            & nuget install vswhere -Version 2.6.7 -SolutionDirectory $solutionDir

```

Comment:
Doesn't work on my machine, please use `StartAndWait` instead.

Reply:
Can do.

So you have more information on how it fails so I can make my scripts more robust in general (for other projects).

Reply:
It says that `nuget` is not known command. I generally recommend using standard commands instead of expression invoke, or at least add obvious strings to quotes.